bdd/spec/021_trig_spec.sh

```diff
@@ -15,33 +15,33 @@ Describe "@std/trig"
         print(trig.log(e));
 
         'Basic Trig functions'.print();
-        print(trig.sin(tau / 6.0));
-        print(trig.cos(tau / 6.0));
-        print(trig.tan(tau / 6.0));
-        print(trig.sec(tau / 6.0));
-        print(trig.csc(tau / 6.0));
-        print(trig.cot(tau / 6.0));
+        print(trig.sin(tau / 6.0 || 0.0));
```

Comment:
I intend to undo all of this and instead edit the `std/trig.ln` file to accept `Result`-wrapped floats.